### ISSUE TITLE
Can't install it on my DS918+

### ISSUE THREAD

**Issue by TheZoker** [runfalk/synology-wireguard#54]
Every time I try to install the latest package, I get this error:
![Screenshot_2020 09 22_18h47m46s_003_](https://user-images.githubusercontent.com/1368405/93912670-53738e80-fd04-11ea-991c-f93de98e9f5b.png)

I already restarted my synology but no change. 

Any idea what causes this? Trust level is set to "Any publisher":
![image](https://user-images.githubusercontent.com/1368405/93912833-8d449500-fd04-11ea-902c-76903f3f492d.png)

**Comment by runfalk** [comment]
Does it work with a previous version?

Did you already have a version installed when you tried this one? I've always uninstalled the old one before upgrading.

**Comment by TheZoker** [comment]
Yes I had a version installed and uninstalled it. Since then I can't install any other version anymore

**Comment by runfalk** [comment]
Curious.

Could you provide log files after you have tried to install? (https://www.synology.com/en-global/knowledgebase/DSM/help/LogCenter/logcenter_search)

Do you have SSH access? If so could you check the contents of `/usr/local/bin/`. The installer creates symlinks there. It would also be nice to see if the `wireguard` folder is still there. It usually installs into one of the volume folder. I don't remember the exact location off the top of my head.

**Comment by TheZoker** [comment]
It seems, that there is no wireguard folder left in `/usr/local/bin/`
![Screenshot_2020 09 23_11h26m36s_005_](https://user-images.githubusercontent.com/1368405/93994112-cf66e880-fd8f-11ea-8995-afcd07163b10.png)

I never actually had wireguard running (yet), so I'm not sure if there is a wireguard folder anywhere 🤔

**Comment by runfalk** [comment]
This is where the wireguard files themselves are installed on my device:

```
$ ls "/volume1/@appstore/WireGuard/wireguard/"
wg  wg-quick  wireguard.ko
```

Can you make sure there are no remnants of such a package?

To be honest I don't really know how to troubleshoot this any further. I assume you have already tried to reboot the device?

**Comment by TheZoker** [comment]
There is no folder left:
![Screenshot_2020 09 23_12h48m21s_006_](https://user-images.githubusercontent.com/1368405/94003137-27571c80-fd9b-11ea-852e-9d4176d4a68d.png)

Yes I also rebooted the device multiple times. Maybe I try to contact synology support for this. Let me know, if there is anything I can try/test 🙂 Thanks anyway!

**Comment by runfalk** [comment]
Technically it could be there on a different volume (like `/volume2`).

If you manage to get in touch with them it'd be helpful to know if there is a way to detect why an SPK fails to install. If the error message is visible somewhere. I do suspect they will be unhelpful since this is a 3rd party package though.

**Comment by Matige** [comment]
The issues is quite old, but if there is still such an error I suggest trying to install the package from CLI. This should give more information about the installation error.
```sh
sudo synopkg install <package.spk>
```
